lib/pay/paddle/webhooks/subscription_payment_succeeded.rb

```diff
@@ -32,6 +34,30 @@ def notify_user(user, charge)
             Pay::UserMailer.receipt(user, charge).deliver_later
           end
         end
+
+        private
+
+        def payment_params(subscription_id)
+          subscription_user = PaddlePay::Subscription::User.list({subscription_id: subscription_id}).try(:first)
+          payment_information = subscription_user ? subscription_user[:payment_information] : nil
+          return {} if payment_information.nil?
+
+          case payment_information[:payment_method]
+          when "card"
+            {
+              card_type: payment_information[:card_type],
+              card_last4: payment_information[:last_four_digits],
+              card_exp_month: payment_information[:expiry_date].split("/").first,
+              card_exp_year: payment_information[:expiry_date].split("/").last
+            }
+          when "paypal"
+            {
+              card_type: "PayPal"
```

Comment:
Do they give you the PayPal email address? I've been storing the paypal email in `card_last4` for Braintree.

Reply:
Unfortunately not: https://developer.paddle.com/api-reference/subscription-api/users/listusers

Reply:
That's too bad, but oh well.